Cubical/Core/Id.agda

```diff
@@ -64,25 +64,30 @@ open import Cubical.Core.PropositionalTruncation public
 
 {- BUILTIN ID Id -}
 
-_≡_ : ∀ {ℓ} {A : Set ℓ} → A → A → Set ℓ
+private
+  variable
+    ℓ ℓ' : Level
+    A : Set ℓ
+    x y z : A
```

Comment:
I would not declare x y z up here as y is used for other things later.

Reply:
Considering that unused/shadowed variables are not gonna be generalized, do you still think so?

I'm fine with writing x y z explicitly.

Reply:
Yes, I'm not a big fan of shadowing the global variables. When reading the file top-down I would get confused if I first think that "y : A" but then y suddenly has another type in some definitions.

Reply:
Works for me.

(BTW if we have the HTML website, people can click on the generalized variable and jump to their `variable` definitions)

Reply:
Done.